FinSearchUnified/Tests/PluginTest.php

```diff
@@ -125,14 +129,39 @@ private function createTestProduct($number, $isActive)
         ];
 
         try {
-            /** @var Article $resource */
-            $resource = $this->manager->getResource('Article');
-            $article = $resource->create($testArticle);
+            $resource = Manager::getResource('Article');
 
-            return $article;
+            return $resource->create($testArticle);
         } catch (Exception $e) {
-            echo sprintf("Exception: %s", $e->getMessage());
+            echo sprintf('Exception: %s', $e->getMessage());
         }
+
+        return null;
+    }
+
+    public function testEmptyValueNotAllowedExceptionIsThrownInExport()
+    {
+        $this->markTestSkipped('Skipping test due to exception class namespace issue');
```

Comment:
We will fix this in the future. For now please ensure that when creating a product with empty values, starting an export will not export the product, but it will be logged in the shopware logs. Please add a screenshot in YouTrack regarding the log line.